lib/RESTClient.js

```diff
@@ -341,6 +341,7 @@ class RESTClient {
      * @return {undefined}
      */
     request(method, beginPath, log, params, data, callback) {
+        const callbackOnce = jsutil.once(callback);
```

Comment:
Idk what stacktrace you've got.

It could be related and fix this ticket: https://scality.atlassian.net/browse/S3C-9411

From what @williamlardier said in this ticket, it could also be due to some `try catch` where the `catch` could call the callback again

Reply:
I doubt it's a similar issue with a `try/catch`, since I don't see any of those in the path leading to the backtrace.

The backtrace that led to the exception is attached in the ticket https://scality.atlassian.net/browse/RD-63.

The present fix ensures that multiple `error` events don't cause multiple calls to the callback in particular, which is definitely something possible (but not 100% sure this is actually what happened in the observed backtrace, although there are good chances it is).